Doctrine/Orm/DataProvider.php

```diff
@@ -107,7 +107,9 @@ public function getCollection($page, array $filters, $itemsPerPage = 30, $order
         }
 
         if ($order) {
-            $queryBuilder->addOrderBy('o.id', $order);
+            foreach ($order as $propName => $value) {
```

Comment:
The project follow semver and is still in `dev-master`, so BC breaks are not a problem for now (btw, I'm finishing a huge refactoring of the `MetadataFactory` that will break BC too).